menu_choice.rb

```diff
@@ -0,0 +1,84 @@
+class MenuChoice
+  def initialize(app)
+    @app = app
+  end
+
+  def execute_action(choice)
+    case choice
+    when 1
+      ListBooks.new(@app).execute
+    when 2
+      ListPeople.new(@app).execute
+    when 3
+      CreatePerson.new(@app).execute
+    when 4
+      CreateBook.new(@app).execute
+    when 5
+      CreateRental.new(@app).execute
+    when 6
+      ListRentalsForPerson.new(@app, self).execute
+    end
+  end
+end
+
+class ListBooks
+  def initialize(app)
+    @app = app
+  end
+
+  def execute
+    @app.list_books
+  end
+end
+
+class ListPeople
+  def initialize(app)
+    @app = app
+  end
+
+  def execute
+    @app.list_people
+  end
+end
+
+class CreatePerson
+  def initialize(app)
+    @app = app
+  end
+
+  def execute
+    @app.create_person
+  end
+end
+
+class CreateBook
+  def initialize(app)
+    @app = app
+  end
+
+  def execute
+    @app.create_book
+  end
+end
+
+class CreateRental
+  def initialize(app)
+    @app = app
+  end
+
+  def execute
+    @app.create_rental
+  end
+end
+
+class ListRentalsForPerson
+  def initialize(app, cli)
+    @app = app
+    @cli = cli
+  end
+
+  def execute
+    person_id = @cli.get_request('Enter person ID: ', :to_i)
+    @app.list_rentals(person_id)
+  end
```

Comment:
When I tried to list the rentals I got the error below. Kindly fix it.

 <img width="501" alt="image" src="https://github.com/Rana-Mudassir/school-library/assets/27709832/6a252b3a-9e2e-4bd9-a806-69234a6859d0">